docs/src/pages/[platform]/connected-components/storage/storagemanager/react.mdx

```diff
@@ -45,7 +45,7 @@ import '@aws-amplify/ui-react/styles.css';
 ```
 </ExampleCode>
 
-At a minimum you must include the `path` and `maxFileCount` props. `path` refers to the S3 image path that will be prefixed to each file key. It is either a `string` or a callback function that accepts the current user's Cognito `identityId` and returns a `string`. See [upload files](https://docs.amplify.aws/react/build-a-backend/storage/upload-files/) 
```

Comment:
This url doesn't exist on the current live docs so replacing it with the gen1 link